doc/users/whats_new/autofmt_xdate.rst

```diff
@@ -0,0 +1 @@
+The autofmt_xdate of the figure.Figure class previously formatted only major tick labels but now formats both minor and major tick labels by passing the argument "which = {None, 'minor', 'major', 'both'}".
```

Comment:
Can you please use the template in `README.rst`?  That makes in very easy to combine each of these files when it comes time for a release.